knojector/steam-authentication-bundle/1.0/config/packages/knojector_steam_authentication.yaml

```diff
@@ -0,0 +1,5 @@
+knojector_steam_authentication:
+    api_key: ''
+    login_route: ''
+    login_redirect: ''
+    user_class: ''
```

Comment:
route, redirect and user_class are all require right?

Reply:
Yes, they are all required. Do I have to make any changes to this?

Reply:
No, you are fine. Thanks